runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/Engine.java

```diff
@@ -206,6 +208,9 @@ public final class Engine implements Collector, AutoCloseable
         schemaTypes.addAll(catalogs.stream().map(Catalog::type).filter(Objects::nonNull).collect(toList()));
         schemaTypes.addAll(models.stream().map(Model::type).filter(Objects::nonNull).collect(toList()));
 
+        final Collection<URL> systemConfigs =
+            exporters.stream().map(Exporter::system).filter(Objects::nonNull).collect(Collectors.toList());
+
```

Comment:
This formatting is a bit easier to read.
```suggestion
        final Collection<URL> systemConfigs = exporters.stream()
            .map(Exporter::system)
            .filter(Objects::nonNull)
            .toList();

```